django/thunderstore/api/cyberstorm/tests/test_packages.py

```diff
@@ -687,3 +688,70 @@ def test_namespace_view__returns_only_packages_listed_in_community_belonging_to_
 
     assert result["count"] == 1
     assert result["results"][0]["name"] == expected.package.name
+
+
+##############################
+# PackageDependantsListAPIView
+##############################
```

Comment:
reminds me of 🍰

any reason we don't create a new file?

obviously not a blocker

Reply:
Files don't grow on (binary) trees, young man.

Anyway my idea was that most of the tests would run against the base class and each implementation class would have like 1 or 2 tests so splitting them seemed unnecessary and possibly give the reader a wrong picture about the test coverage of the implementations. I'm not sure how valid this is anymore after the base class swallowed out the community-scoping parts.